sharepoint_utils.py

```diff
@@ -0,0 +1,40 @@
+from tika import parser
+
+
+def print_and_log(logger, level, message):
+    print(message)
+
+    if level.lower() == 'debug':
+        log_debug(logger, message)
+    elif level.lower() == 'info':
+        log_info(logger, message)
+    elif level.lower() == 'warn':
+        log_warn(logger, message)
+    elif level.lower() == 'error':
+        log_error(logger, message)
+
+
+def log_debug(logger, message):
```

Comment:
you are working too much here. your `print_and_log` function can just be:

```
def print_and_log(logger, level, message):
    getattr(logger, level.lower())(message)
```

and you can remove all those small functions